shell/src/main/java/alluxio/cli/fsadmin/report/MetricsCommand.java

```diff
@@ -11,44 +11,44 @@
 
 package alluxio.cli.fsadmin.report;
 
-import alluxio.client.meta.MetaMasterClient;
+import alluxio.client.metrics.MetricsMasterClient;
 import alluxio.grpc.MetricValue;
-import alluxio.metrics.ClientMetrics;
-import alluxio.metrics.MasterMetrics;
-import alluxio.metrics.MetricsSystem;
-import alluxio.metrics.WorkerMetrics;
 import alluxio.util.FormatUtils;
 
+import com.google.common.math.DoubleMath;
+
 import java.io.IOException;
 import java.io.PrintStream;
 import java.text.DecimalFormat;
 import java.text.DecimalFormatSymbols;
 import java.util.Locale;
 import java.util.Map;
-import java.util.TreeMap;
+import java.util.SortedSet;
+import java.util.TreeSet;
 
 /**
  * Prints Alluxio metrics information.
  */
 public class MetricsCommand {
+  private static final String BYTES_METRIC_IDENTIFIER = "Bytes";
+  private static final String THROUGHPUT_METRIC_IDENTIFIER = "Throughput";
   private static final DecimalFormat DECIMAL_FORMAT
       = new DecimalFormat("###,###.#####", new DecimalFormatSymbols(Locale.US));
-  private static final String INDENT = "    ";
+  private static final String INFO_FORMAT = "%s  (Type: %s, Value: %s)%n";
```

Comment:
Do we want to use tabs or something for pretty alignment?

Reply:
We have no categories in the new way of printing all the metrics. Previously indent is used for separating category name and its values. So I remove the indent.